.github/workflows/goth.yml

```diff
@@ -0,0 +1,80 @@
+name: Goth (PR and push)
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      # - <your-branch>    # put your branch name here to test it @ GH Actions
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  goth-tests:
+    name: Run integration tests
+    runs-on: goth
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Configure python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8.0'
```

Comment:
I know this was used elsewhere but it kinda bothers me that we're specifying a version with a `0` patch number... shouldn't this be just `3.8`? or the _last_ patch number for 3.8 ?